chain/client/src/sync/state.rs

```diff
@@ -31,6 +31,7 @@ use near_async::messaging::CanSendAsync;
 use near_chain::chain::ApplyStatePartsRequest;
 use near_chain::near_chain_primitives;
 use near_chain::resharding::StateSplitRequest;
+use near_chain::types::RuntimeAdapter;
```

Comment:
havent finished testing it out on a localnet yet, but did you mean to just add all this state sync stuff? looks like it wasnt there before and maybe it was something you had added to the git index by accident?

Reply:
Accident, fixed.